stylesheets/commons/Fountain.less

```diff
@@ -460,6 +460,14 @@ The smaller sizes are for sub-sections. */
 	text-shadow: 0.0625rem 0.0625rem 0.125rem #000000;
 }
 
+.facetDesc {
+	border-top: 0.156rem solid #505050;
```

Comment:
1. 0.156rem = 2.496px, I prefer to have that be a rounded number. So either 0.125rem (2px) or 0.1875rem (3px).

2. As hjpalpha mentioned, would be nice if you could use one of these grey variables:
![image](https://github.com/user-attachments/assets/9c33c888-6dff-44d2-ab18-a6e809575734)
If you're unsure on how to view all available variable: you can inspect the website and then look at the right side of the devtools, scroll down and then you see this: 
![image](https://github.com/user-attachments/assets/c4b14625-de53-4657-bc50-cad861d63bf9)
Click on the link that is marked with red and then you see the whole list.

3. Also provide a dark theme variable if needed. For example:
```
.facetDesc {
  border-top: 0.156rem solid #505050;
  
  .theme--dark & {
    border-color: var( --clr-moon-90)
  }
}

```

Reply:
Thank you for the feedback!

I'll add a dark theme variable for the outer box border, and have the facetDesc to be a rounded number with the var colors suggested by hjpa.